content/en/apps/payments/payments-api.md

```diff
@@ -0,0 +1,145 @@
+---
+date: '2020-01-08T09:59:25Z'
+menu:
+  apps:
+    parent: "payments"
+    name: Corda Payments API
+title: Corda Payments API
+weight: 400
+---
+Use the Payment Service API to initiate payments from a node on a Corda network.
+
+The methods listed below are triggered by flows within Corda.
+
+
+## `makePayment`
+
+Initiate a payment between participating nodes on the network.
+
+### Parameters
+
+* `amount`: Amount<`FiatCurrency`> - the amount and currency of the payment to make.
```

Comment:
`FiatCurrency` is a pretty weird little type. We should either summarise the definition somewhere on this page or link to it (even though we didnt invent it for CP, but from another library)